knx/dpt/types_registry.go

```diff
@@ -224,6 +225,7 @@ func setup() {
 
 			// Convert the name into KNX yy.xxx (e.g. DPT_1001 --> 1.001) format
 			name = name[4:len(name)-3] + "." + name[len(name)-3:]
+			name = strings.Replace(name, "141.200", "14.1200", -1) // Workaround for DPT_141200
```

Comment:
Is there another way than to replace ad-hoc?

Reply:
The problem is how the naming of the DPTs is done.  DPT_14000.
as long as the subtype has only 3 digits you just have to count beginn from the end.

But now there are also subtypes with 4 digits.

So u dont know if 141200 means 141.200 or 14.1200

Another way is to change the naming of the ALL DPTs.   =>  DPT_14_000 instead of DPT_14000.

Then you know the part after the second "_" belongs to the subtype.

Reply:
Maybe do it like [here](https://github.com/mobilarte/knx-exp/blob/main/knx/dpt/types_registry.go), no guessing which format to convert, simply map it.

Reply:
> Maybe do it like [here](https://github.com/mobilarte/knx-exp/blob/main/knx/dpt/types_registry.go), no guessing which format to convert, simply map it.

I dont get it. What u mean?

Reply:
I also prefer explicit naming in this case.